tests/vmi_multus_test.go

```diff
@@ -856,14 +864,19 @@ var _ = Describe("[Serial]SRIOV", func() {
 			waitVmi(vmi1)
 			waitVmi(vmi2)
 
+			updatedVmi1, err := virtClient.VirtualMachineInstance(vmi1.Namespace).Get(vmi1.ObjectMeta.Name, &metav1.GetOptions{})
```

Comment:
vmi.Name would be enough i think, no need ObjectMeta 
same all over please

Reply:
Done.